root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -1787,3 +1776,118 @@ test('Cleanup', function (t) {
 
     t.end();
 });
+
+test('URL cleanup component: auto-select, clean-up, and validation', {}, function (t) {
+    const test_data = [
+        // AllMusic
+        {
+            i_url: 'http://www.allmusic.com/album/here-comes-the-sun-mw0002303439/releases',
+            i_ent: 'release_group',
+            match: 'allmusic',
+            clean: 'http://www.allmusic.com/album/mw0002303439',
```

Comment:
It's confusing to use a different naming system for inputs and outputs. :( Why not `o_rel` instead of `match` and `o_url` instead of `clean`? Or using 'e_' (for expected). Having a comment in the source code (rather than just in the commit message) that explains what these mean would also be welcome.